### ISSUE TITLE
Improve control over segmentation output filenames

### ISSUE THREAD

**Issue by mathieuboudreau** [ivadomed/ivadomed#906]
### Motivation for the feature
I'm trying to implement IVADOMED in AxonDeepSeg, and would like for consistency that our output masks filenames remain the same pre/post IVADOMED implementation.

See: https://github.com/neuropoly/axondeepseg/pull/547

### Description of the feature
It would be nice to have more control over the segmentation output filenames.

Currently I see that I can control the "suffix" (note the quotation marks, as "_pred" always seems to be added) of the individual binary masks in the model file (at least, in the json).

<img width="205" alt="Screen Shot 2021-08-05 at 10 22 02 PM" src="https://user-images.githubusercontent.com/1421029/128442004-3013a9b1-7dc9-4f29-b648-82766c7cf531.png">

<img width="402" alt="Screen Shot 2021-08-05 at 10 22 17 PM" src="https://user-images.githubusercontent.com/1421029/128442001-d86b16f2-3d83-487e-9977-d6592eaf2111.png">

I'd really like it if I could omit the "_pred" part that's added somewhere in the code.

Also, it looks like the combined mask of the binary images is set to always be called "mask" (dot) (file extension). It would be great to have more control over this filename, as 1) I don't see it set anywhere in the config files of the models and 2) we already have a file called "mask.png" in a lot of our datasets, as some of them were manually segmented and called "mask.png" to be used in training.

**Comment by mathieuboudreau** [comment]
Oh wait, maybe I confused myself actually; does IVADOMED even output mask.png? Looks like no right? I think I confused myself as it was already in my folder; then only my first point would stand (remove the `_pred` suffix after the user-input suffix in the model file)

**Comment by mariehbourget** [comment]
You are right, the file called `mask.png` is not an output of ivadomed but is the original mask from ADS.
The 2 files generated by ivadomed are `image_seg-axon-manual_pred.png` and `image_seg-axon-manual_pred.png`.

Currently in ivadomed, there are no direct options to control the output filenames.
The "_seg-axon-manual" and "_seg-myelin-manual" `target_suffix` refers to the names of the ground-truths used during training and testing and are not meant to be changed to control the output filenames.

The` _pred` part is added automatically to differenciate the ground-truth (`<image_name>_seg-axon-manual.png`) from the prediction (`<image_name>_seg-axon-manual_pred.png`).

In ADS, we will have to create the "combined" masks from these outputs with ADS suffix naming convention (`_seg-axonmyelin`). Similarly, I think it would be best to rename the 2 files outputted by ivadomed in ADS to follow our convention (`_seg-axon`, `_seg-myelin`).

Otherwise, I think it would be possible to remove the `_pred` part in ivadomed but the `-manual` part would remain and we would have to rename them anyway.

**Comment by mariehbourget** [comment]
Hi @mathieuboudreau, I forgot to ask in the meeting this morning, is the suggestion I made above to rename ivadomed's output PNG files in ADS ok with you?

**Comment by mathieuboudreau** [comment]
> Hi @mathieuboudreau, I forgot to ask in the meeting this morning, is the suggestion I made above to rename ivadomed's output PNG files in ADS ok with you?

Sounds good to me! I did implement the `_seg-axonmyelin` merged mask on our side; we'll just have to change some of the default suffix keys I think also.
